packages/react-reconciler/src/__tests__/ReactFiberHostContext-test.internal.js

```diff
@@ -13,16 +13,20 @@
 let React;
 let ReactFiberReconciler;
 let ConcurrentRoot;
+let DefaultEventPriority;
 
 describe('ReactFiberHostContext', () => {
   beforeEach(() => {
     jest.resetModules();
     React = require('react');
     ReactFiberReconciler = require('react-reconciler');
-    ConcurrentRoot = require('react-reconciler/src/ReactRootTags');
+    ConcurrentRoot = require('react-reconciler/src/ReactRootTags')
```

Comment:
The changes in this file aren't related. The tests were already wrong because this import was assigning the whole default exports object to `ConcurrentRoot`, instead of just that tag. So the test below was passing an invalid value as a root tag, which caused to to behave as if it were in legacy mode.

This PR added a check for `root.tag === LegacyRoot`, which exposed the mistake in this file.